scripts/README.md

```diff
@@ -0,0 +1,544 @@
+    C -->|Start This Prefect Flow| D[ python orchestration/flows/bl832/alcf.py ]
+    D -->|Step 1: Transfer from data832 to ALCF| E[ /eagle/IRIBeta/als/bl832/raw/< experiment folder name convention>/< h5 files>]
+    E -->|Step 2: Run Globus Flows on ALCF Globus Compute Endpoint| F[ A: Run reconstruction.py <br> B: Run tiff_to_zarr.py]
+    F -->|Save Reconstruction on ALCF| G[ /eagle/IRIBeta/als/bl832/scratch/< experiment folder name convention>/rec< dataset>/< tiffs> <br> /eagle/IRIBeta/als/bl832/scratch/< experiment folder name convention>/rec< dataset>.zarr/ ]
+    G -->|Step 3: Transfer back to data832| J[ /data/scratch/globus_share/< experiment folder name convention>/rec< dataset>/< tiffs> <br> /data/scratch/globus_share/< experiment folder name convention>/rec< dataset>.zarr/ ]
+    J -->|Schedule Pruning| L[Prune raw and scratch data from ALCF and data832]
+    J --> M[Visualize Results] 
+    J -->|SciCat| N[TODO: ingest metadata into SciCat]
+    M -->|2D Image Slices| O[view .tiff images ]
+    M -->|3D Visualization| P[load .zarr directory in view_tiff.ipynb]
+
+	classDef storage fill:#A3C1DA,stroke:#A3C1DA,stroke-width:2px,color:#000000;
+	class C,E,G,J storage;
+
+	classDef collection fill:#D3A6A1,stroke:#D3A6A1,stroke-width:2px,color:#000000;
+	class A,B collection;
+
+	classDef compute fill:#A9C0C9,stroke:#A9C0C9,stroke-width:2px,color:#000000;
+	class D,F,K,L,N compute;
+
+	classDef visualization fill:#E8D5A6,stroke:#E8D5A6,stroke-width:2px,color:#000000;
+	class M,O,P visualization;
+```
+
+## File flow details
+**Naming conventions**
+-   Experiment folder name:
+	- `< Proposal Prefix>-< 5 digit proposal number>_< first part of email address>/`
+-   h5 files in experiment folder:
+	- `< YYYYMMDD>_< HHMMSS>_< user defined string>.h5`
+-   h5 files for tiled scans:
+	- `< YYYYMMDD>_< HHMMSS>_< user defined string>_x<##>y<##>.h5`
+
+**data832 Raw Location**
+- `data/raw/< Experiment folder name convention>/< h5 files>`
+
+**data832 Scratch Location**
+- `data/scratch/globus_share/< Experiment folder name convention>/rec< dataset>/< tiffs>`
+- `data/scratch/globus_share/< Experiment folder name convention>/rec< dataset>.zarr/`
+
+**ALCF Raw Location**
+- `/eagle/IRIBeta/als/bl832/raw/< Experiment folder name convention>/< h5 files>`
+
+**ALCF Scratch Location**
+- `/eagle/IRIBeta/als/bl832/scratch/< Experiment folder name convention>/rec< dataset>/< tiffs>`
+- `/eagle/IRIBeta/als/bl832/scratch/< Experiment folder name convention>/rec< dataset>.zarr/`
+
+## Globus Compute Flow
+
+The preferred method for scheduling jobs on ALCF is through Globus Compute. This `reconstruction_wrapper()` function contains the code registered with Globus, which runs the reconstruction on ALCF. The commands call two Python programs located in the IRIBeta project folder on Polaris. Here is an example of code that is registered and run in the Globus Compute Flow:
```

Comment:
Rather than copying code here, maybe just link to the file? Just concerned about this copy getting outdated.

Reply:
What file is this referring to?